pyne/tally.pyx

```diff
@@ -1,17 +1,20 @@
 ################################################
 #                 WARNING!                     #
 # This file has been auto-generated by xdress. #
-# Do not modify!!!                             #
 #                                              #
 #                                              #
-#                    Come on, guys. I mean it! #
+#                                              #
+# With some edits from BaM241                  #
 ################################################
 """
 """
 from pyne cimport stlcontainers
 from libc.stdlib cimport free
 from libcpp.map cimport map as cpp_map
 from libcpp.string cimport string as std_string
+from libcpp.vector cimport vector
+import numpy as np
+cimport numpy as np
```

Comment:
The `numpy` is imported here twice.

Reply:
one is an `import` the other one a `cimport` used by Cython.

I am not sure why this is used in `material.pyx` and seems to be normal according to [Cython docs](https://cython.readthedocs.io/en/latest/src/userguide/sharing_declarations.html)